src/box/lua/config/configdata.lua

```diff
@@ -445,6 +445,37 @@ local function validate_failover(found, peers, failover, leader)
     end
 end
 
+-- Verify that the given replicaset contains at least one
```

Comment:
Nit: commit message typo: "must be in read-only mode"

Reply:
Thanks! Fixed.